darwin/Platform.c

```diff
@@ -114,6 +116,8 @@ int Platform_numberOfFields = LAST_PROCESSFIELD;
 
 double Platform_timebaseToNS = 1.0;
 
+long Platform_clockTicksPerSec = -1;
```

Comment:
There's no need to initialize this to -1.

Reply:
This value is used in case someone inadvertedly calls `DarwinProcess_ticksToNanoseconds` before `Platform_init` …

Reply:
Hmm.  Platform_init is called quite early during startup, there's no chance of that IMO - i.e. we haven't initialized the entire platform but we're already making per-process timing calculations?  (IMO *that* would be the bug to fix, not initializing the variable to -1 instead of the default zero).

But, not a big deal either way, -1 is OK too.

Reply:
Agreed, calling in that weird order would definitively a bug worth fixing.
But so is potentially causing a div-by-zero … ;-)
Let's go with it as is for now.